_posts/2019-06-06-NeuralODEs.markdown

```diff
@@ -0,0 +1,289 @@
+---
+layout: post
+title:  "Neural ODEs"
+date:   2019-06-06 10:00:00 -0400
+categories: neural-nets
+author: luca
+blurb: "Neural ODEs are neural network models which generalize standard layer to layer propagation to continuous depth models. The forward propagation in many neural networks models can be seen as one step of discretation of an ODE; starting from this observation, one can construct and efficiently train models by numerically solving ODEs. On top of furnishing a new whole family of architectures, neural ODEs can be applied to gain memory efficiency in supervised learning tasks, construct a new class of invertible density models and model continuous time-series"
```

Comment:
"... Starting from the observation that the forward propagation in neural networks is equivalent to one step of discretation of an ODE, we can construct and efficiently train models by via ODEs. On top of providing a novel family of architectures, notably for invertible density models and continuous time series, neural ODEs also provide a memory efficiency gain in supervised learning tasks."

Reply:
changed